sirepo/template/madx.py

```diff
@@ -159,6 +159,18 @@ def write_files(self, data, source_path, dest_dir):
         return PKDict()
 
 
+class _MadxLogParser(template_common.LogParser):
+    def _parse_log_line(self, line):
+        res = ""
+        if re.search(r"^\++ (error|warning):", line, re.IGNORECASE):
+            line = re.sub(r"^\++ ", "", line)
+            res += line + "\n"
+        elif re.search(r"^\+.*? fatal:", line, re.IGNORECASE):
+            line = re.sub(r"^.*? ", "", line)
+            res += line + "\n"
+        return res
+
```

Comment:
Looks much better. In cases like the above, you don't need to use the ```res``` variable because you get at most one error from the line. I saw similar code in the opal.py module:
```python
         if re.search(r"^\++ (error|warning):", line, re.IGNORECASE):
            return re.sub(r"^\++ ", "", line)
        if re.search(r"^\+.*? fatal:", line, re.IGNORECASE):
            return re.sub(r"^.*? ", "", line)
        return None
```